src/study.js

```diff
@@ -80,8 +81,16 @@ export default async function runStudy(devMode) {
             Glean.setUploadEnabled(false);
           }
       });
+      
+      // send the study enrollment ping to the secure analysis environment.
+      const uid = devMode ? "00000000-0000-0000-0000-000000000000" : rally._rallyID;
+      if (!devMode && !uid) {
+        console.error("Rally ID not acquired by study.");
+      }
+      studyEnrollmentMetrics.id.set(uid);
+      rs01Pings.studyEnrollment.submit();
```

Comment:
It's not a problem if this is called before `Glean.initialize`, just know that everything that happens before that call is batched and will be executed after Glean initializes :-) See [our docs](https://mozilla.github.io/glean/book/reference/general/initializing.html#behavior-when-uninitialized)